docusaurus/docs/iOS/ReferenceDocs/Sources/StreamChat/APIClient/Array.md

```diff
@@ -1,7 +1,8 @@
 ---
 id: array 
 title: Array
---- 
+slug: ReferenceDocs/Sources/StreamChat/APIClient/array
+---
```

Comment:
This should be:
```
---
id: array
title: Array
slug: reference-docs/sources/streamchat/apiclient/array
```
and make sure it works locally (local url is the same as slug)